src/components/loading-indicator/__tests__/makeLoadable.test.tsx

```diff
@@ -19,7 +20,11 @@ describe('components/loading-indicator/makeLoadable', () => {
 
     test('should pass props down to BaseComponent', () => {
         const LoadableComponent = makeLoadable(TestComponent);
-        const wrapper = shallow(<LoadableComponent className="foo" hello="123" isLoading={false} />);
+        const props = {
+            className: 'foo',
+            hello: '123',
+        };
+        const wrapper = shallow(<LoadableComponent isLoading={false} {...props} />);
```

Comment:
putting className and hello directly as props causes TS error. Something with the makeLoadable HOC.